mission-vision-values.md

```diff
@@ -0,0 +1,100 @@
+and observability vendors have created differentiation – and lock-in – through
+agents that automatically gather useful telemetry from a wide integration
+surface. This lock-in is limiting for customers and inhibits innovation by
+tightly coupling telemetry collection and telemetry storage and analysis. With
+OpenTelemetry, we strive to provide a level playing field for all observability
+providers, avoiding lock-in for any vendor and interoperating with other OSS
+projects in the telemetry and observability ecosystem.
+
+**Opportunity IV: Telemetry should be loosely coupled.** OpenTelemetry end-users
+should be able to pick and choose from the pieces they want without bringing in
+the rest of the project. To enable this, OpenTelemetry’s software architecture
+is decoupled wherever possible. As a corollary, this also means that
+OpenTelemetry does not want to “pick winners” when it comes to particular
+projects or technologies: where possible, we prefer to create optionality for
+our end-users.
+
+**Opportunity V: Telemetry should be built-in.** Historically, telemetry was
+something developers integrated manually or via post-compilation agents.
+OpenTelemetry believes that high-quality telemetry can be built in to the
+entire software stack – just like testing has been for decades.
+
+While the structure and technical details of OpenTelemetry may change over
+time, these five key opportunities will remain outstanding until we achieve our
+mission, and as a project we refer to them to orient – and reorient – as we
+chart our path.
+
+## Engineering Values (the principles that guide our contributions)
+
+OpenTelemetry’s mission and vision describe where we want to go.
+OpenTelemetry’s engineering values describe how we want to get there.
+
+OpenTelemetry’s core engineering values are Compatibility, Stability,
+Resiliency, and Performance.
+
+### We Value *Compatibility*
+
+With the number of stakeholders and supported platforms, following
+specifications and enabling interoperability is very important. This means that
+OpenTelemetry strives to be standards-compliant, vendor-neutral, and consistent
+across languages and components.
+
+### We Value *Stability*
+
+As many libraries take dependencies on OpenTelemetry APIs, API stability and
+backwards compatibility is vital for our end-users, and thus for our work. As a
+corollary, we do not introduce new concepts unless we’re confident they’re
+needed by a broad subset of OpenTelemetry’s end-users.
+
+### We Value *Resilience*
+
+In OpenTelemetry we value technical resiliency: the ability to adapt to
+continue operating even in the face of resource scarcity or other environmental
+challenges. OpenTelemetry is designed to work and keep collecting telemetry
+signals when an application is misbehaving, and OpenTelemetry code is designed
+to degrade gracefully as needed.
+
+### We Value *Performance*
+
+The cardinal rule in observability is “first, do no harm”: to that end,
```

Comment:
"do no harm" to me also means "don't crash my app", which is not "performance", and not exactly the same as "resilience" above

Reply:
Perhaps we could merge these sections

Reply:
Related: in the face of constraints, business logic should always take precedence over telemetry collection.

Reply:
let's not merge these sections. Performance is intentionally value number 4, less important than 3 ahead of it. Maybe just rephrase that it is more about ROI of telemetry than "no harm".